### PR TITLE
Fix migration for next of kin

### PR DESCRIPTION
**Pull request by LindaKadz**
Dropped the table for next of kin

Created a new table for next of kin with the correct data type.

### CODE REVIEW

**Review by 75pollet — COMMENTED** [review]
The code is okay but indentation is not uniform.

File: lib/nailinda_web/router.ex [hunk 1]
```diff
@@ -18,6 +18,8 @@ defmodule NailindaWeb.Router do
 
     get "/patient/new", PatientController, :new
     get "/", PageController, :index
+    get "/patient/:id", PatientController, :edit
```
Comment (okothkongo) [inline lib/nailinda_web/router.ex]: how about you use patient resource

File: lib/nailinda/user/user.ex [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+	end
+
     def get_all_patients do
         Patient
     	|> Repo.all
     end
 
     def get_patient_by_id(id) do
-         Repo.get(Patient, id)
+			Patient
```
Comment (okothkongo) [inline lib/nailinda/user/user.ex]: check on the identification

**Review by okothkongo — COMMENTED** [review]
how about you add gif to show the operation your current changes on the feature

File: lib/nailinda_web/router.ex (hunk 1 above)
Comment (LindaKadz) [reply lib/nailinda_web/router.ex]: it shall be refactored once delete has been merged, not on this branch

File: lib/nailinda/user/user.ex (hunk 2 above)
Comment (LindaKadz) [reply lib/nailinda/user/user.ex]: already fixed it

**Pull request merged**